libs/cardano-ledger-core/src/Cardano/Ledger/State/StakePool.hs

```diff
@@ -132,6 +137,8 @@ data StakePoolState = StakePoolState
   -- ^ Optional metadata (URL and hash)
   , spsDeposit :: !(CompactForm Coin)
   -- ^ Deposit for each pool
+  , spsDelegs :: !(Set (Credential 'Staking))
```

Comment:
I don't think we are saving much by shortening, but by expanding the name we are reducing ambiguity.
```suggestion
  , spsDelegators :: !(Set (Credential 'Staking))
```